### PR TITLE
blackman

### PR DESCRIPTION
**Pull request by ndurumo254**

### CODE REVIEW

**Comment by ndurumo254** [comment]
https://github.com/unifyai/ivy/issues/11197

**Comment by ndurumo254** [comment]
https://github.com/unifyai/ivy/issues/11190

**Comment by ndurumo254** [comment]
https://github.com/unifyai/ivy/issues/11011

**Comment by ndurumo254** [comment]
https://github.com/unifyai/ivy/issues/8422

**Comment by ndurumo254** [comment]
https://github.com/unifyai/ivy/pull/11129

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @ndurumo254 

Sorry for the late review🙂
Some few changes to be made then we progress also remember to use pre-commit🤣
I'm getting some unnecessary tabs introduced in the manipulation file

Thanks

**Comment by ndurumo254** [comment]
> Hey @ndurumo254
> 
> Sorry for the late review🙂 Some few changes to be made then we progress also remember to use pre-commit🤣 I'm getting some unnecessary tabs introduced in the manipulation file
> 
> Thanks

ok.

File: ivy/functional/frontends/jax/numpy/manipulations.py [hunk 1]
```diff
@@ -151,3 +151,49 @@ def vsplit(ary, indices_or_sections):
...
 def hsplit(ary, indices_or_sections):
     return ivy.hsplit(ary, indices_or_sections)
+
+
+@to_ivy_arrays_and_back
+def bartlett(M):
+    if M == 1:
+        return ivy.ones(1)
+    elif M % 2 == 0:
+        return ivy.concatenate(
```
Comment (hirwa-nshuti) [inline ivy/functional/frontends/jax/numpy/manipulations.py:161]: `ivy.concatenate` is not present in ivy API but, it is available as `ivy.concat`

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_manipulation.py [hunk 2]
```diff
@@ -863,3 +863,85 @@ def test_jax_numpy_hsplit(
...
+    fn_tree="jax.numpy.blackman",
+    dtype_and_m=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        min_num_dims=2,
+    ),
+    test_with_out=st.just(False),
+)
+def test_jax_numpy_blackman(
+    *,
+    dtype_and_M,
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_manipulation.py:907]: Try to pass generated data correctly as this is supposed to be `dtype_and_m`

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey

The unit tests are still failing would you look into them😊
And also do check some comments I added below

Thanks

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey I see that still your unit tests are failing now

Would you have a look and fix them😊

**Comment by ndurumo254** [comment]
i want to finalize on this

**Comment by hirwa-nshuti** [comment]
Sure

**Pull request closed without merging**